### DISCUSSION TITLE
Stripe Webhook/Endpoint

### DISCUSSION THREAD

**Discussion by dathwa** [OFFLINE-GmbH/oc-mall-plugin#1110] [Q&A] [answered]
Hello. I am connecting Mall with Stripe.
There is an option in stripe to add a Webhook/Endpoint.
Is this necessary? If so, what url do I use?
Thanks

**Comment by tobias-kuendig** [comment, marked as answer]
@dathwa The webhook is not required as Mall already knows about the payment outcome when the user returns to the site during checkout. 

You can get some more details about the payments  (and some payment methods) using the webhooks, but you would have to implement this yourself, as it is not part of the Mall plugin's feature set.
